templates/docker-compose/Makefile

```diff
@@ -4,6 +4,7 @@ include .env
 
 DOCKER_BIN=docker
 DOCKER_COMPOSE_BIN=docker-compose
+GIT_BIN=git
 COMPOSE_FILES=-f docker-compose.yml -f docker-compose.prod.yml
```

Comment:
What happens if you need to add `-f docker-compose.override.yml`?

Reply:
should just recommend in docs that they do that on a separate file that is not tracked in repo